File: a2m-afp-back/src/main/java/egovframework/com/a2m/egov/model/TccoFile.java
package egovframework.com.a2m.egov.model;

import java.io.Serializable;
import java.util.Date;

import com.fasterxml.jackson.annotation.JsonInclude;

import io.swagger.annotations.ApiModel;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import personal.aug.convert.MapAndObjectConversion;
import personal.aug.convert.annotations.MapKey;

/**
 * @author KetHX
 */
@Getter
@Setter
@NoArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
@ApiModel(description = "Model File")
public class TccoFile extends MapAndObjectConversion implements Serializable {

	private static final long serialVersionUID = 1L;

	@MapKey("ATCH_FLE_SEQ")
	private String atchFleSeq;

	@MapKey("FLE_KEY")
	private String fleKey;

	@MapKey("FLE_TP")
	private String fleTp;

	@MapKey("FLE_PATH")
	private String flePath;

	@MapKey("FLE_NM")
	private String fleNm;

	@MapKey("NEW_FLE_NM")
	private String newFleNm;

	@MapKey("FLE_SZ")
	private String fleSz;

	@MapKey("CREATED_BY")
	private String createdBy;

	@MapKey("CREATED_DATE")
	private Date createdDate;

	@MapKey("UPDATED_BY")
	private String updatedBy;

	@MapKey("UPDATED_DATE")
	private Date updatedDate;

	private String createdDateStr;

	private String updatedDateStr;

}

File: a2m-afp-front/src/model/tccofile.ts
export class TccoFile {
    atchFleSeq: string | null = "";
    fleKey?: string | null;
    fleTp?: string | null;
    flePath?: string | null;
    fleNm?: string | null;
    newFleNm?: string | null;
    fleSz?: string | null;
    createdBy?: string | null;
    createdDate?: any | null;
    updatedBy?: string | null;
    updatedDate?: string | null;

    public fromMap(data: any): any {
        if (!data) return null;
        this.atchFleSeq = data['ATCH_FLE_SEQ'];
        this.fleKey = data['FLE_KEY'];
        this.fleTp = data['FLE_TP'];
        this.flePath = data['FLE_PATH'];
        this.fleNm = data['FLE_NM'];
        this.newFleNm = data['NEW_FLE_NM'];
        this.fleSz = data['FLE_SZ'];
        this.createdBy = data['CREATED_BY'];
        this.createdDate = data['CREATED_DATE'];
        this.updatedBy = data['UPDATED_BY'];
        this.createdDate = data['UPDATED_DATE'];

        return this;
    }

    public toMap(): any {
        if (!this) return null;

        return {
            'ATCH_FLE_SEQ': this.atchFleSeq,
            'FLE_KEY': this.fleKey,
            'FLE_TP': this.fleTp,
            'FLE_PATH': this.flePath,
            'FLE_NM': this.fleNm,
            'NEW_FLE_NM': this.newFleNm,
            'FLE_SZ': this.fleSz,
            'CREATED_BY': this.createdBy,
            'CREATED_DATE': this.createdDate,
            'UPDATED_BY': this.updatedBy,
            'UPDATED_DATE': this.updatedDate
        };
    }

    public fromObject(data: any): TccoFile {
        if (!data) return this;
        Object.assign(this, data);

        return this;
    }

    constructor(data?: any) {
        Object.assign(this, data);
    }
}